Vagrantfile

```diff
@@ -65,18 +73,22 @@ Vagrant.configure(2) do |config|
   # Unfortunately, VirtualBox only allows to share absolute paths, which cannot
   # work across all host OS.
   # https://www.virtualbox.org/ticket/15305
-  #config.vm.synced_folder ".", gconfig['vm_shared_folder'], owner: gconfig['vm_user']
+  # config.vm.synced_folder ".", gconfig['vm_shared_folder'], owner: gconfig['vm_user']
   
   # Disable the default shared folder of vagrant
   config.vm.synced_folder ".", "/vagrant", disabled: true
-   
+
   # provisioner: set up VM via ansible. To (re-)run this step:
   #   vagrant provision --provision-with ansible
+  # Note we use a static inventory, see: https://www.vagrantup.com/docs/provisioning/ansible_intro#static-inventory
+  config.vm.network :private_network, ip: gconfig["ansible_host"]
   config.vm.provision "ansible" do |ansible|
+    ansible.inventory_path = "inventory.yml"
+    ansible.limit = "vagrant-provision"
+    ansible.playbook = "playbook-build.yml"
     ansible.verbose = "v"
```

Comment:
since this will be the main output people see, I suggest we comment this (in most cases, verbose output should not be needed)

Reply:
yeh makes sense 👍